metabase/metabase
# Add basic raw data picker for app scaffolding

Replaces the `DataSelector` component we've been using for app scaffolding with a custom component built from scratch for data app needs.

Two key differences are that we're using a two-pane layout similar to the model/saved question picker + allow multiple table selections which `DataSelector` can't handle.

Next steps: add search functionality, allow picking models and not only raw data.

### To Verify

1. New > App
2. Play around with picking different databases, schemas, and tables. Select/unselect stuff and ensure nothing weird happens
3. Make sure you can scaffold an app with a different set of tables
4. While being on the app, click "+" in the app navigation bar, pick "Data"
5. Make sure the data picker there works the same

### Demo

https://user-images.githubusercontent.com/17258145/196510847-50bf7681-66a8-4499-beda-0742da5253fb.mp4
